index.js

```diff
@@ -1,10 +1,14 @@
-var keypather = module.exports = function (opts) {
+module.exports = factoryAndInstance;
+function factoryAndInstance (opts) {
   var keypather = new Keypather(opts && opts.force);
   return keypather;
 };
+Keypather.call(factoryAndInstance);
+factoryAndInstance.__proto__ = new Keypather().__proto__;
 
-function Keypather (force) {
-  this.force = (force !== undefined) ? Boolean(force) : true; // force - default: true
+function Keypather (opts) {
+  if (!(this instanceof Keypather)) { return new Keypather(opts); }// Auto instantiate
+  this.force = (opts && typeof opts.force !== 'undefined' && Boolean(opts.force) === false ? false : true);
```

Comment:
I don't think you need these modifications just revert it to what it was
`this.force = (force !== undefined) ? Boolean(force) : true; // force - default: true`